.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
+}
```

Comment:
Can we remove this? Probably want to add `.vscode` to `.gitignore` in a later PR